extension/content/script.js

```diff
@@ -87,17 +89,41 @@ async function initUI() {
 
   $("#region-select").addEventListener("change", reloadEngines);
   $("#locale-select").addEventListener("change", reloadEngines);
+  $("#distro-id").addEventListener("input", reloadEngines);
+  $("#engine-id").addEventListener("change", calculateLocaleRegions);
+  $("#locale-by-engine").addEventListener("change", calculateLocaleRegions);
 
   $("#reload-engines").addEventListener("click", reloadEngines);
   $("#reload-page").addEventListener("click", reloadPage);
 
   $("#engines-table").addEventListener("click", showConfig);
```

Comment:
So looks like this is broken by this PR, probably needs to to be a lazy binding, it also doesnt work in the by-engines view

Reply:
I've just fixed this, it was actually because I'd dropped the line-height css and the calculation was busted.

However, I have also fixed the new view to handle needing reloading, e.g. when you decide to change something part way through a run.